oaknational/Oak-Web-Application
# feat: blog pagination in query params

## Description

- Adds pagination to query params in blog indenx page
- fixes a11y complaint on blog categories list (that there was no label for the `nav`)

## Issue(s)

Fixes #530 

## How to test

1. Go to {cloud link}
2. Click through pagination (all blogs and category pages)
3. You should see the url changing
4. If you change the url manually it should change the page too
**note well: if you come to a page=3 (f.i.) hard, then you'll be shown a flash of page 1 first ... this is because we are statically generating pages. When we're not, we'll be able to have the server serve up the correct page**


## Checklist
